UKHomeOffice/sgmr-service
# SPCR-513 Redirect to landing page when not logged in

### AC / Description of changes

- for invalid URLs redirect to landing page
- when user not logged in and attempts to visit valid url, redirect to landing page


### To Test

- be signed out
- go to /people
- > you should be taken to the landing page
- sign in
- go to /asdfasdfdsaf
- > you should be taken to the landing page


### Notes

Requirement is that we redirect to the GOV UK starting page, from there they can click start which will either prompt signin, or take them to the dashboard and they can continue

---

**Remember to merge to feature branches not `master`.**
